services/src/registry.ts

```diff
@@ -18,6 +19,7 @@ async function run() {
   await opaHelper.initializeForRegistry();
 
   logger.info('Stitch registry booting up...');
+  await loadPlugins();
```

Comment:
Maybe we want to add a flag here to load only plugin types that are relevant to registry?

Reply:
Loading plugins is just creation of object array.
The only thing we can skip in registry is `buildArgumentInjectionGlobals`.
I'm not sure it worth adding flag.